src/demos/index.ts

```diff
@@ -108,9 +109,66 @@ export default {
     DlBarChartDemo,
     DlLineChartDemo,
     DlSpinner,
-    DlConfusionMatrix,
-    DlToggleButtonDemo,
     DlEllipsisDemo,
+    DlThumbnailGallery
+}
+
+export const demoState = {
```

Comment:
We removed demoState in change for a export default a long time ago. please revert this to match.

Reply:
I solved it